### ISSUE TITLE
Update all builds to use Nodejs 10.19.0.2 or newer

### ISSUE THREAD

**Issue by nickboldt** [eclipse-che/che#16599]
### Describe the bug

See CVE issue https://issues.redhat.com/browse/CRW-809

TL;DR is we should stop using ancient versions of nodejs 10 with vulernabilities.

In git@github.com:eclipse/che-theia.git I see:

```
** [1] ./generator/tests/production/assembly/yarn.lock ** 
340-"@types/node@*":
341-  version "10.7.1"
342:  resolved "https://registry.yarnpkg.com/@types/node/-/node-10.7.1.tgz#b704d7c259aa40ee052eec678758a68d07132a2e"

344-"@types/node@8.10.20":

** [2] ./yarn.lock ** 
2184-"@types/node@10.12.10":
2185-  version "10.12.10"
2186:  resolved "https://registry.yarnpkg.com/@types/node/-/node-10.12.10.tgz#4fa76e6598b7de3f0cb6ec3abacc4f59e5b3a2ce"
--
2194-"@types/node@^10.0.0":
2195-  version "10.17.19"
2196:  resolved "https://registry.yarnpkg.com/@types/node/-/node-10.17.19.tgz#1d31ddd5503dba2af7a901aafef3392e4955620e"
```

We should move up to 10.19.0 or newer.

In Dockerfiles we seem to be OK:

```
** [1] ./dockerfiles/theia-dev/docker/ubi8/builder-image-from.dockerfile ** 
1:# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/nodejs-10
2:FROM registry.access.redhat.com/ubi8/nodejs-10:1-66

** [2] ./dockerfiles/theia/docker/ubi8/runtime-from.dockerfile ** 
1:# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/ubi8/nodejs-10
2:FROM registry.access.redhat.com/ubi8/nodejs-10:1-66 as runtime

** [3] ./dockerfiles/theia-endpoint-runtime-binary/Dockerfile ** 
11:FROM quay.io/eclipse/che-custom-nodejs-deasync:10.19.0-latest as custom-nodejs
```

But we should move to newer UBI:

* registry.access.redhat.com/ubi8/nodejs-10:1-66.1584015429

### Che version

 - [x] latest (master branch of che-theia)
 - [ ] nightly
 - [ ] other: please specify

**Comment by benoitf** [comment]
@nickboldt the `@types/**` are nothing to do with the runtime
it's like declaration of interfaces.

**Comment by benoitf** [comment]
TL;DR there is no security issue
